public/class-amazonpolly-public.php

```diff
@@ -296,15 +296,27 @@ private function include_audio_player( $language_code, $audio_location, $autopla
 		if($common->is_tim_limitless_enabled()){
             $post_id = $GLOBALS['post']->ID;
             $php_config=array();
+            if(!$common->is_excerpt_adder_enabled()){
```

Comment:
extract this block to function,
write it more readable (maybe switch-case or some kind of hash-table).

the row of
$postHash = get_post_meta( $post_id, TIM_LIMITLESS_POST_HASH_CONTENT_TITLE, true );
can be written once. you just need the function to return the key-name-const